packages/kbn-test/src/page_load_metrics/cli.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Url from 'url';
+
+import { run, createFlagError } from '@kbn/dev-utils';
+import { capturePageLoadMetrics } from './capture_page_load_metrics';
+
+export function runPageLoadMetricsCli() {
+  run(
+    async ({ flags, log }) => {
+      log.info('!!! Do not forget to load data with index pattern !!!');
```

Comment:
@spalger it might be not obvious, but when we run `node scripts/page_load_metrics.js` we don't load data with esArchiver. 
When puppeteer navigates to `app/discover`, it is auto-redirected to `app/kibana#/management/kibana/indexPatterns` and we actually got duplicated events for `app/discover` in [responsesByPageView](https://github.com/elastic/kibana/pull/66224/files#diff-99651f21ea952b96a2f1586c7dca69bcR24) map and assetSizeMeasurements may contain +1 extra value in array.

I'm not sure if adjusting filter to remove the duplicate is a good idea, maybe printing message is good enough?

Reply:
I think we should probably add validation for each app path that looks for some specific element to indicate that the page has loaded correctly and throws an error if the app doesn't load correctly, rather than trying to support scenarios where the correct apps don't load

Reply:
I was also thinking this in relation to the log line you added to the CLI, ideally we should focus on the specific problems that pop up rather than trying to remind people to load a specific esArchive. If they run this script against a cloud instance there's no reason we should need or recommend an index pattern.

Reply:
I added validation to check unique element on each page:

```
ERROR Page '/app/discover' was not loaded properly, current url: http://localhost:5620/app/kibana
ERROR UNHANDLED ERROR
ERROR TimeoutError: waiting for selector "[data-test-subj="discover-sidebar"]" failed: timeout 60000ms exceeded
          at new WaitTask (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/DOMWorld.js:388:34)
          at DOMWorld._waitForSelectorOrXPath (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/DOMWorld.js:313:26)
          at DOMWorld.waitForSelector (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/DOMWorld.js:296:21)
          at Frame.waitForSelector (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/FrameManager.js:384:51)
          at Frame.<anonymous> (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/helper.js:95:27)
          at Page.waitForSelector (/Users/dzmitrylemechko/github/kibana/node_modules/puppeteer/lib/Page.js:778:33)
          at Promise.all.apps.map (/Users/dzmitrylemechko/github/kibana/packages/kbn-test/src/page_load_metrics/navigation.ts:118:18)
          at process._tickCallback (internal/process/next_tick.js:68:7)
```